Sources/SwiftKafka/RDKafka/RDKafkaTopicConfig.swift

```diff
@@ -0,0 +1,54 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the swift-kafka-gsoc open source project
+//
+// Copyright (c) 2022 Apple Inc. and the swift-kafka-gsoc project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of swift-kafka-gsoc project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Crdkafka
+
+/// A collection of helper functions wrapping common `rd_kafka_topic_conf_*` functions in Swift.
+struct RDKafkaTopicConfig {
```

Comment:
Can we apply the same here with the class instead of struct approach.

Reply:
(See comment about `RDKafkaConfig`)